src/util/std_expr.h

```diff
@@ -131,35 +115,31 @@ class symbol_exprt:public exprt
   }
 };
 
-/*! \brief Expression to hold a symbol (variable)
-*/
+/// Expression to hold a symbol (variable)
 class decorated_symbol_exprt:public symbol_exprt
 {
 public:
   decorated_symbol_exprt()
   {
   }
 
-  /*! \brief Constructor
-   * \param identifier Name of symbol
-  */
+  /// Constructor
+  /// \param identifier Name of symbol
   explicit decorated_symbol_exprt(const irep_idt &identifier):
     symbol_exprt(identifier)
   {
   }
 
-  /*! \brief Constructor
-   * \param  type Type of symbol
-  */
+  /// Constructor
```

Comment:
I think 'Constructor' is obvious by the section doxygen puts it in, no?

Reply:
Yes, but I figured it was useful to keep the param descriptions (and therefore the "Constructor" title). What do you prefer?

Reply:
Can't you describe params without a method description? Anyways, that's not very important whether there is a description or not. My comment was just about the usefulness of the current description which suggests that we should rather remove it.

Reply:
Yes, you can have only the param descriptions. So I removed the `/// Constructor` lines.